runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -212,25 +219,43 @@ protected void startUp() throws Exception
 
 		final BufferedImage iconImage = ImageUtil.getResourceStreamFromClass(getClass(), "screenshot.png");
 
+		popup.put("Open screenshot folder...", () ->
+		{
+			try
+			{
+				Desktop.getDesktop().open(SCREENSHOT_DIR);
+			}
+			catch (IOException ex)
+			{
+				log.warn("Error opening screenshot dir", ex);
+			}
+		});
+
+		if (config.uploadScreenshot() == UploadStyle.NEITHER)
```

Comment:
I don't see why this would need to require having upload style set to neither considering its a right click option. Also this wouldn't work anyway since the config is only checked on startup.

Reply:
Removed it